rules/windows/persistence_webshell_detection.toml

```diff
@@ -0,0 +1,61 @@
+[metadata]
+creation_date = "2021/08/24"
+maturity = "production"
+updated_date = "2021/08/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious commands executed via a web server, which may suggest a vulnerability and remote shell access.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Webshell Detection"
```

Comment:
Because there are so many approaches, we may want to be more specific here

```suggestion
name = "Webshell Detection: Script Process Child of Common Web Processes"
```

Or similar. Thoughts?